### ISSUE TITLE
Error: no such option: --counts-data

### ISSUE THREAD

**Issue by 10adavis** [Teichlab/cellphonedb#258]
After specifying "--counts-data=gene_name", I am receiving the message: "Error: no such option: --counts-data"

This seems to be a problem specific to running on Windows 10 using the Anaconda Powershell prompt, as it works fine when running on Ubuntu. Any way to specify --counts-data for Windows?

**Comment by dienerpatrick** [comment]
I have the exact same problem!

**Comment by Lovelock1998** [comment]
I have the exact same problem......

**Comment by prete** [comment]
This seems to be a known Windows issue, are you both on Windows as well? 
Could you also provide the full command you're using to launch CellPhoneDB, please?

**Comment by bhavyaac** [comment]
Not sure if this will help anyone in this thread, but I would like to share my experience with this error. I am using a Linux system (on a compute cluster), not Windows.

**TL;DR it may help to check that your CellphoneDB version is correct, and that your dependencies have installed properly.**

I got this error, and it turns out that I had some sort of issue with `rpy2` installation (like other Windows users had experienced e.g. #184), and I believe this caused CellphoneDB version 1.1.0 to be installed instead of 2.1.7 (like in issue #338). This makes sense, because I believe the `--counts-data` argument was not introduced yet in version 1.1.0. 
I solved this by ensuring that my python and pip paths were correct i.e. within my virtual environment (e.g. `which pip`), and that you have at least Python 3.6 (run `python --version`, and the virtual environment was also created in Python >= 3.6). 
Once I had checked this, I installed the desired version of CellphoneDB with `pip install cellphonedb==2.1.7 --no-deps` in order to avoid the dependency conflict from before. Then, I installed each line of [requirements.txt](https://github.com/Teichlab/cellphonedb/blob/master/requirements.txt) individually using `pip install "boto3>=1.7,<1.7.99"` for example. (Note: I had to install `Cython` before installing `scikit-learn`.) Now, I was able to run CellphoneDB 2.1.7 normally.
I hope this makes sense. I'm a novice to virtual environments and python installations, so I would appreciate feedback as well!
